src/App.tsx

```diff
@@ -96,7 +109,7 @@
               },
             )}
           >
-            <div className="tile is-child box is-success ">
+            <div className="tile is-child box is-success">
               {selectedPost && <PostDetails post={selectedPost} />}
```

Comment:
The condition `selectedPost && <PostDetails post={selectedPost} />` correctly displays post details when a post is selected. Ensure that `selectedPost` is being set accurately when a post is selected.